File: primo-anno/Programmazione/python/cumulative_sum.py
#creare una funzione che fa una somma cumulativa dei numeri in input fin quando l'utente non digita 0

somma = 0


while True: 
    
    try:    
        addendo = int(input("inserisci un numero: "))

    except ValueError: 
        print("mongoloide ho detto numero, NUMERO, N-U-M-E-R-O, sai leggere o sei semplicemente ritardato?")



    if addendo != 0: 
        somma += addendo
    else: 
        break


print(somma)

File: primo-anno/Programmazione/C/cumulative_sum.c
/*scrivere una funzione in C per calcolare la somma di un insieme di numeri inseriti da tastiera. il programma deve
- fermarsi quando viene inserito 0
-leggere una sequenza di numeri 
-visualizzare il valore corrispondente alla somma dei valori introdotti*/
#include <stdio.h>



int main(){

    int input; 
    int somma = 0; 

    while (input != 0){
        printf("inserisci un numero "); 
        scanf("%d", &input); 

        somma+=input;
        

    }

    printf("%d", somma); 
     
}
